### PR TITLE
Add new operators and extractors deep dive blog post.

### PR DESCRIPTION
**Pull request by denimalpaca**
Signed-off-by: Benji Lampel <benjamin@astronomer.io>

### CODE REVIEW

File: contents/blog/operators-and-extractors-technical-deep-dive/index.mdx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+title: How Operators and Extractors Work Under-the-Hook
+date: 2022-08-01
+author: Benji Lampel
+image: ./image.svg
```
Comment (rossturk) [inline contents/blog/operators-and-extractors-technical-deep-dive/index.mdx:5]: I sent you some artwork that you can use for this, and for banner.svg.

File: contents/blog/operators-and-extractors-technical-deep-dive/index.mdx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+banner: ./banner.svg
+description: A technical deep-dive on how the Airflow OSS and OpenLineage OSS projects interact. 
+---
+
+## Overview
+
+Airflow Operators and OpenLineage Extractors have a specific, if quirky, way of working together. Recently, the way they work together has seen a bit of an overhaul, and the new SQL Check Extractors added a new and unique way that the extractors work and interact with operators. In this blog, we'll demystify these relationships.
+Note: This blog post describes the relationships of the operators and extractors only for `Airflow>=2.3` and `OpenLineage>=0.12.0`.
```
Comment (rossturk) [inline contents/blog/operators-and-extractors-technical-deep-dive/index.mdx:13]: This Markdown renderer is collapsing this into a single paragraph without a blank line in between - this happens throughout.

**Review by rossturk — CHANGES_REQUESTED** [review]
Awesome post! Need some artwork + bit of formatting, then we are gtg.

**Review by merobi-hub — APPROVED** [review]
LGTM!

**Review by rossturk — APPROVED** [review]

**Comment by denimalpaca** [comment]
@rossturk @merobi-hub any reason this hasn't been merged yet?

**Pull request merged**